actors/builtin/market/market_actor.go

```diff
@@ -610,6 +620,11 @@ func (a Actor) CronTick(rt Runtime, params *adt.EmptyValue) *adt.EmptyValue {
 			rt.Abortf(exitcode.ErrIllegalState, "failed to flush deal states: %s", err)
 		}
 
+		st.Proposals, err = proposals.Root()
+		if err != nil {
+			rt.Abortf(exitcode.ErrIllegalState, "failed to flush deal proposals: %s", err)
```

Comment:
Use `RequireNoError` (in lots of places above where the error is unexpected).

Reply:
This is done.